docs-next/components/Navigation.tsx

```diff
@@ -61,6 +61,11 @@ export const Navigation = () => {
           Testing
         </NavItem>
       </Section>
+      <Section label="Tutorials">
```

Comment:
Can we shift this section above `Guides`? I think we probably want to have `Getting Started` as one of the first things people are drawn to.

Reply:
Sure thing... :) I also don't think we should have all Tutorials in the nav... More like a landing page for Tutorials and that's it
If that happens we may want to keep it where it is now.
In fact @timleslie what do you think about removing it for now and just adding the link to the getting started tut into the top section of guides?

Reply:
Or we have it look like this later down the track?

<img width="494" alt="image" src="https://user-images.githubusercontent.com/1266923/111926778-4ba17500-8b02-11eb-981a-e1e2112a11f8.png">

Reply:
Hmm, let's shift it up for now, since I think for now it looks good, and revisit the decision once we have a couple more tutorials. Hopefully by then some of the other pieces of the puzzle will have fallen into place and the correct answer will present itself.